docs/manual/guides/install-test-versions.de.md

```diff
@@ -0,0 +1,151 @@
+Release Candidates oder Entwicklerversionen installiert werden. Nach der Änderung 
+der `composer.json` muss natürlich eine komplette Paketaktualisierung durchgeführt 
+werden. 
+
+
+## Release Candidates installieren
+
+Release Candidates nutzen eine spezifische Form von sogenannten »Release-Tags«.
+Der erste Release Candidate von Contao `4.9` heißt z. B. `4.9.0-RC1`. Mit Hilfe
+der »[Version Requirement Syntax][composerVersions]« von Composer können wir verlangen,
+dass solche Versionen ebenfalls installiert werden sollen, und nicht nur die stabilen
+Versionen. Diese Angabe erfolgt z. B. mit `4.9@RC`, und muss bei _allen_ Bundles
+des Contao Cores angewandt werden, _inklusive_ dem `contao/core-bundle` und dem
+`contao/installation-bundle`, welche normalerweise nicht direkt in der eigenen `composer.json`
+festgelegt werden.
+
+Hier ist ein komplettes Beispiel um die neueste Contao `4.9` Version installieren
+zu lassen, _inklusive_ den neuesten Release Candidates (wenn vorhanden):
+
+```json
+{
+    "name": "contao/managed-edition",
+    "type": "project",
+    "description": "Contao Open Source CMS",
+    "require": {
+        "php": "^7.1",
+        "contao/calendar-bundle": "^4.9@RC",
+        "contao/comments-bundle": "^4.9@RC",
+        "contao/conflicts": "@dev",
+        "contao/core-bundle": "^4.9@RC",
+        "contao/faq-bundle": "^4.9@RC",
+        "contao/installation-bundle": "^4.9@RC",
+        "contao/listing-bundle": "^4.9@RC",
+        "contao/manager-bundle": "4.9.*@RC",
+        "contao/news-bundle": "^4.9@RC",
+        "contao/newsletter-bundle": "^4.9@RC"
+    },
+    "conflict": {
+        "contao-components/installer": "<1.3"
+    },
+    "extra": {
+        "contao-component-dir": "assets",
+        "symfony": {
+            "require": "^4.2"
+        }
+    },
+    "scripts": {
+        "post-install-cmd": [
+            "Contao\\ManagerBundle\\Composer\\ScriptHandler::initializeApplication"
+        ],
+        "post-update-cmd": [
+            "Contao\\ManagerBundle\\Composer\\ScriptHandler::initializeApplication"
+        ]
+    }
+}
+```
+
+Beachte dass dabei später auch die stabilen Versionen installiert 
+werden (wie z. B. `4.9.0`, `4.9.1` etc.), wenn eine Paketaktualisierung durchgeführt wird.
+
+
+## Entwicklerversionen installieren
+
+Während der Release Candidate und davor auch während der Entwicklungsphase einer
+Contao Version kann auch die Entwicklerversion zum Testen installiert werden. Auf
+diese Weise können die neuesten Änderungen sofort getestet werden, ohne auf einen
+neuen Release Candiate warten zu müssen. Natürlich kann dies auch instabilen Programmcode
+enthalten.
+
+Statt einer spezifischen _Version_ wird nun ein spezifischer _Branch_ aus dem öffentlichen
+Git Repository von Contao verlangt. Jede »Minor« Version von Contao hat ihren eigenen
+Entwicklungs-Branch, z. B. `4.9.x-dev` für Contao `4.9`. Der Branch für die neueste,
```

Comment:
```suggestion
Entwicklungszweig, z. B. `4.9.x-dev` für Contao `4.9`. Der Branch für die neueste,
```

Reply:
"Branch" würde ich eigentlich nirgendwo übersetzen. Sonst müssten wir ja auch

> wir nun ein spezifischer __Zweig__ aus dem öffentlichen Git __Verwahrungsort__

schreiben.

Reply:
Das hatte eigentlich einen anderen Grund: Mir gefällt der Bindestrich in "Entwicklungs-Branch" nicht besonders.

Reply:
In der Deutschen Rechtschreibung setzt man unter Anderem dann einen Bindestrich, wenn ein Wort aus verschiedenen Wortarten besteht. Ich denke, dass das auch für Wörter gilt, die aus verschiedenen Sprachen bestehen. Siehe zB https://www.business-schreibkurse.de/deutsch-englische-wortkombinationen-mit-oder-ohne-bindestrich/

Reply:
> Siehe zB https://www.business-schreibkurse.de/deutsch-englische-wortkombinationen-mit-oder-ohne-bindestrich/

Die dort angeführten Beispiele sind alle korrekt. In unserem Fall hier geht es aber vor allem um den [Fugenlaut (Fugen-s)](https://de.wikipedia.org/wiki/Fugenlaut), wo das Vorderwort der Genitiv­form gleicht und ein Bindestrich meist überflüssig ist.

Zur besseren Übersichtlichkeit und Lesbarkeit _kann_ jedoch ein Bindestrich verwendet werden, insbesondere, wenn die Komposita relativ kompliziert und lang sind: http://journalismusausbildung.de/fugen-s-und-bindestrich

Aber egal, das ist alles nicht so wichtig! :)

Reply:
Ok, ja verstehe.